hxformat.json

```diff
@@ -8,6 +8,7 @@
 		}
 	},
 	"sameLine": {
+		"ifBody": "same",
 		"ifElse": "next",
```

Comment:
Also, wouldn't these conflict in styles? LMAO

Reply:
something on the haxe library for lime
![image](https://user-images.githubusercontent.com/85742902/142060803-17ac608e-c2ac-4141-9a4a-6df93c6a4e71.png)

Reply:
> 
> 
> something on the haxe library for lime ![image](https://user-images.githubusercontent.com/85742902/142060803-17ac608e-c2ac-4141-9a4a-6df93c6a4e71.png)

This isn't the Haxe library for Lime, though?

Reply:
but its basically the same thing
same file name,
same format,
so i wanted to update it here too